containers/backend/backend/app/routers/colormap.py

```diff
@@ -43,18 +43,23 @@ async def get_colormap(
 
     ::kwarg num_values int Number of values to generate in the colormap
     """
+    if stretch_range is None:
+        stretch_range_arg: list[int] = [0, 10]
```

Comment:
It was a quick guess based on the datasets I'm familiar with! Should we fail with a 400 bad request?

Reply:
Some 4xx error for sure - doesn't FastAPI already return the appropriate error if we remove `Optional` from the parameter and it's missing from the request?
I'd say that not having a default sounds safer, just because the value is purely dependent on the data domain so in most cases it should be explicitly provided anyway - and presumably if some client code was forming requests incorrectly (e.g. with a typo in the `stretch_range` parameter name), this could lead to confusing behaviour where the default would be used and the tileserver would return tiles with incorrect colours but that would be difficult to spot.